lib/s3routes/routes.js

```diff
@@ -134,9 +136,10 @@ function checkTypes(req, res, params, logger) {
  *  values for whether queries are supported
  * @param {function} params.dataRetrievalFn - function to retrieve data
  * @param {RequestLogger} logger - werelogs logger instance
+ * @param {String} [s3config] - s3 configuration
```

Comment:
Is this actually optional? It looks required to me.

Reply:
Yes optional.